<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JLKBKOP" failures="1" tests="2" name="Error Validation" time="33.089" errors="0" timestamp="2023-01-09T07:08:04 EST">
  <testcase classname="testNGframeWork.test.ErrorValidation" name="LoginerrorValidation" time="0.964"/>
  <testcase classname="testNGframeWork.test.ErrorValidation" name="ProducterrorValidation" time="3.381">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGframeWork.test.ErrorValidation.ProducterrorValidation(ErrorValidation.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- ProducterrorValidation -->
</testsuite> <!-- Error Validation -->
